src/client/client.test.ts

```diff
@@ -305,4 +320,27 @@ describe('Merge path with `app.route()`', () => {
     type verify = Expect<Equal<boolean, typeof data.ok>>
     expect(data.ok).toBe(true)
   })
+
+  describe('Multiple endpoints', () => {
+    const api = new Hono()
+      .get('/foo', (c) => c.jsonT({ foo: '' }))
+      .post('/bar', (c) => c.jsonT({ bar: 0 }))
+    const app = new Hono().route('/api', api)
+    type AppType = typeof app
+    const client = hc<typeof app>('http://localhost')
+
+    it('Should return correct types - GET /api/foo', async () => {
+      type verify1 = Expect<NotEqual<typeof client.api.foo.$get, any>>
```

Comment:
Why would you assert that `typeof client.api.foo.$get` is `NotEqual` to `any`? In the next line, you get `res` with the correct type, so I suppose you can compare `typeof client.api.foo.$get` with that type, can't you?

Reply:
Ah, you are right, it's not necessary. At first, I've been checked only `typeof client.api.foo.$get`, so I didn't write the assert for `typeof data.foo`. But if we assert the `typeof data.foo`, we don't need to check `typeof client.api.foo.$get`.

Reply:
Removed. Thanks for pointing.